### PR TITLE
Revamp input

### PR DESCRIPTION
**Pull request by nhat-ndm1193**

### CODE REVIEW

**Comment by nhat-ndm1193** [comment]
Screenshot 2022-01-25 at 10 00 40 AM

File: src/components/atoms/Input/Component.tsx [hunk 1]
```diff
@@ -21,104 +21,95 @@ const renderText = (content: any, defaultProps: any, style: StyleProp<any>) =>
...
+      disabled,
+      inputContainerStyle,
+      inputStyle,
+      errorStyle,
+      errorMessage,
+      label,
+      labelStyle,
+      showClearText,
+      iconStyle,
+      multiline,
```
Comment (zrg-team) [inline src/components/atoms/Input/Component.tsx:51]: numberOfLines > 1 is multiline. It simpler for

File: src/components/atoms/Input/Component.tsx [hunk 2]
```diff
@@ -21,104 +21,95 @@ const renderText = (content: any, defaultProps: any, style: StyleProp<any>) =>
...
+const Input: FunctionComponent<InputProps> = forwardRef<any, InputProps>(
+  (
+    {
+      containerStyle,
+      disabled,
+      inputContainerStyle,
+      inputStyle,
+      errorStyle,
+      errorMessage,
+      label,
```
Comment (zrg-team) [inline src/components/atoms/Input/Component.tsx:47]: please use props name: "style" as much as possible

**Review by zrg-team — CHANGES_REQUESTED** [review]

**Comment by zrg-team** [comment]
@nhat-ndm1193 
https://www.notion.so/iruuzainc/All-components-JSON-structure-a275f7c1b6aa4aab91907b8f7c7d3a35

File: src/components/atoms/Input/Component.tsx (hunk 2 above)
Comment (zrg-team) [reply src/components/atoms/Input/Component.tsx:37]: @nhat-ndm1193 can it text or title which defined in our BE ?

Comment (nhat-ndm1193) [reply src/components/atoms/Input/Component.tsx:37]: @zrg-team yah lemme check

**Review by zrg-team — APPROVED** [review]

**Pull request merged**